dbms/src/Interpreters/InterpreterSelectQuery.cpp

```diff
@@ -131,6 +138,9 @@ InterpreterSelectQuery::~InterpreterSelectQuery() = default;
 
 void InterpreterSelectQuery::init(const Names & required_result_column_names)
 {
+    /// Failpoint to make our query begin after the write action finish.
+    FAIL_POINT_PAUSE(FailPoints::pause_query_until_write_finish);
```

Comment:
Actually, by enabling this fail point, it cannot do what its name is telling with: pause until write finish. I would prefer to prefix something like `internal_` and rename `force_pause_...` to `pause_...` instead. What do you think?